daemon/main.go

```diff
@@ -451,7 +448,25 @@ func initEnv() {
 	config.Opts.Set(endpoint.OptionConntrack, !disableConntrack)
 	config.Opts.Set(endpoint.OptionConntrackAccounting, !disableConntrack)
 	config.Opts.Set(endpoint.OptionConntrackLocal, false)
-	config.Opts.Set(endpoint.OptionPolicy, enablePolicy)
+
+	switch config.EnablePolicy {
```

Comment:
I think we want a `config.EnablePolicy = strings.ToLower(config.EnablePolicy)` here before the switch statement